pom.xml

```diff
@@ -58,8 +58,13 @@
                         </goals>
                         <configuration>
                             <bnd><![CDATA[
-Bundle-Name: ${project.name} (No embedded dependencies)
+Bundle-Name: ${project.name} (Framework Fragment, No embedded dependencies)
 Bundle-SymbolicName: ${project.artifactId}.nodeps
+ExtensionBundle-Activator: org.apache.sling.javax.activation.internal.Activator
+Export-Package: javax.activation;version="0.0.0",com.sun.activation.registries;version="0.0.0",com.sun.activation.viewers;version="0.0.0"
```

Comment:
This will embed. I would rather clean what is exported by the system bundle (https://lists.apache.org/thread.html/r0aa3c02c38b5026e24684a00de3c6c8506e166334e77892ac2ee0ab3%40%3Cdev.sling.apache.org%3E)

Reply:
Hi @kwin, my updated pr will not embed any more thanks to your link to https://groups.google.com/forum/#!topic/bndtools-users/NAdU0_X8OKY for the bnd instructions.

I agree that updating what is exported by the system bundle is a good idea. However I would still like to have the exports as defined by this PR in the bundle/fragment as I'd like to be able to run it with already released versions of Sling.